src/test/resources/net/bramp/ffmpeg/fixtures/ffprobe-big_buck_bunny_720p_1mb.mp4

```diff
@@ -4,34 +4,40 @@
             "index": 0,
```

Comment:
Instead of changing `fixtures/ffprobe-big_buck_bunny_720p_1mb.mp4`, can we add a new `fixtures/ffprobe-side_data_list` ? 

`fixtures/ffprobe-side_data_list` can be similar to fixtures/ffprobe-big_buck_bunny_720p_1mb.mp4 but then we don't need to change the `samples/ffprobe-big_buck_bunny_720p_1mb.mp4`, and you do not need to change `FFMpegExecutorTest.java` and `ReadmeTest.java`.

Reply:
I think you're right. Thank you for your opinion.